kubernetes/elastic-helm/configmap-daemonset.yaml

```diff
@@ -0,0 +1,265 @@
+          labels:
+            - tag_name: app.label.component
+              key: app.kubernetes.io/component
+              from: pod
+    receivers:
+      filelog:
+        retry_on_failure:
+          enabled: true
+        start_at: end
+        exclude:
+        # exlude collector logs
+        - /var/log/pods/default_otel-daemonset-opentelemetry-collector-agent*_*/opentelemetry-collector/*.log
+        include:
+        - /var/log/pods/*/*/*.log
+        include_file_name: false
+        include_file_path: true
+        operators:
+        - id: container-parser
+          type: container
+      hostmetrics:
+        collection_interval: 10s
+        root_path: /hostfs
+        scrapers:
+          cpu:
+            metrics:
+              system.cpu.utilization:
+                enabled: true
+              system.cpu.logical.count:
+                enabled: true
+          memory:
+            metrics:
+              system.memory.utilization:
+                enabled: true
+          process:
+            mute_process_exe_error: true
+            mute_process_io_error: true
+            mute_process_user_error: true
+            metrics:
+              process.threads:
+                enabled: true
+              process.open_file_descriptors:
+                enabled: true
+              process.memory.utilization:
+                enabled: true
+              process.disk.operations:
+                enabled: true
+          network:
+          processes:
+          load:
+          disk:
+          filesystem:
+            exclude_mount_points:
+              mount_points:
+                - /dev/*
+                - /proc/*
+                - /sys/*
+                - /run/k3s/containerd/*
+                - /var/lib/docker/*
+                - /var/lib/kubelet/*
+                - /snap/*
+              match_type: regexp
+            exclude_fs_types:
+              fs_types:
+                - autofs
+                - binfmt_misc
+                - bpf
+                - cgroup2
+                - configfs
+                - debugfs
+                - devpts
+                - devtmpfs
+                - fusectl
+                - hugetlbfs
+                - iso9660
+                - mqueue
+                - nsfs
+                - overlay
+                - proc
+                - procfs
+                - pstore
+                - rpc_pipefs
+                - securityfs
+                - selinuxfs
+                - squashfs
+                - sysfs
+                - tracefs
+              match_type: strict
+      kubeletstats:
+        auth_type: serviceAccount
+        collection_interval: 20s
+        endpoint: ${env:K8S_NODE_NAME}:10250
+        node: '${env:K8S_NODE_NAME}'
+        # Required to work for all CSPs without an issue
+        insecure_skip_verify: true
+        k8s_api_config:
+          auth_type: serviceAccount
+        metrics:
+          k8s.pod.cpu.node.utilization:
```

Comment:
Maybe we can also enable the [k8s.pod.memory.node.utilization](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.105.0/receiver/kubeletstatsreceiver/documentation.md#k8spodmemorynodeutilization)

Reply:
Ah! Saw it now, sure I will create a follow-up PR. Thanks!